docs-ref-conceptual/run-azure-cli-docker.md

```diff
@@ -70,6 +71,24 @@ After halting any containers running the CLI image, remove it.
 docker rmi mcr.microsoft.com/azure-cli
 ```
 
+## Microsoft Copilot for Azure CLI
```

Comment:
Please remove this H2 and PNG. Copilot for Azure is not directly related to running the Azure CLI in a Docker Container.